backoffice/templates/backoffice/home.html

```diff
@@ -3,6 +3,24 @@
 {% block heading %}Dashboard{% endblock %}
 
 {% block main %}
+  {% if drivers_pending > 0 or accounts_pending > 0 %}
+    <a href="/backoffice/approvals/" class="bg-yellow-300 flex-row hover:bg-yellow-100 m-2 p-2">
+      ❗
+      {% if accounts_pending > 0 %}
+        <div class="inline-block">
+          {% if accounts_pending > 1 %}
+            {{accounts_pending}} billing accounts require approval
+          {% else %}
+            1 billing account requires approval
+          {% endif %}
+        </div>
+      {% endif %}
+      {% if drivers_pending > 0 %}
+        <div class="inline-block">{{drivers_pending}} driver requires approval</div>
+      {% endif %}
+      ❗
+    </a>
+  {% endif %}
```

Comment:
The backoffice home page is one of the few backoffice pages that currently works OK (ish) on mobile devices. Please can you check that this new change also works OK on portrait mobile (just resizing a browser window to be really narrow should be enough)?

Reply:
Changed font size to text-sm, this means it should always fit on one line on phones.